flow/scripts/variables.yaml

```diff
@@ -292,6 +292,11 @@ SYNTH_RETIME_MODULES:
     optimal distribution of registers on long pipelines. See OR discussion #8080.
   stages:
     - synth
+DFF_MAP_FILE:
+  description: |
+    List of D flip flops treated as a black box by Yosys.
```

Comment:
Wrong description, I see all other *_MAP_FILE descriptions are wrong too. I propose "Optional mapping file supplied to Yosys to map D flip-flops."